examples/src/main/kotlin/examples/onnx/cv/efficicentnet/EfficientNet4Lite.kt

```diff
@@ -23,7 +23,7 @@ import java.io.File
  * This examples demonstrates the inference concept on EfficientNet4Lite model:
  * - Model configuration, model weights and labels are obtained from [ONNXModelHub].
  * - Model predicts on a few images located in resources.
- * - Special preprocessing (used in EfficientNet4Lite during training on ImageNet dataset) is applied to images before prediction.
+ * - Special preprocessing (used in EfficientNet4Lite during training on ImageNet dataset) is applied to image before prediction.
```

Comment:
Why "image" is better than "images"? "each image" also seems to fit here. (same for other places)